index.qmd

```diff
@@ -53,127 +53,145 @@ bibliography: references.bib
 
-Third. Based on the idea that networks follow a trophic hierarchy and that network structure can be determined by distributing interactions along single dimension (the “niche axis”, @allesinaGeneralModelFood2008). Essentially these models can be viewed as being based on the idea of resource partitioning (niches) along a one-dimensional resource which will result in the standard 'trophic pyramid' to ensure that all species can 'fit' along this resource (which has strong ties back to the idea of intervality) *e.g.,* Cascade model [@cohenCommunityFoodWebs1990], Niche model [@williamsSimpleRulesYield2000], Nested hierarchy model [@cattinPhylogeneticConstraintsAdaptation2004].
+The second set of networks are constructed based on a probabilistic rule-set about diet choice and niche breadth, reflecting fundamental ideas of foraging biology *e.g.,* consumers feeding on resources that are smaller than themselves. **RATHER THAN ALLESINA, I'D SUGGEST PRESENTING THE 5 RULES IN THE ADBM EXPLANATION OF THE NICHE MODEL** These models that are based on the compartmentation and acquisition of energy for species at different trophic levels [@allesinaFoodWebModels2009; @krauseCompartmentsRevealedFoodweb2003] and that network structure can be determined by distributing interactions along single dimension [the 'niche axis'; @allesinaGeneralModelFood2008], while parametrising an aspect of the network structure [although see @allesinaFoodWebModels2009 for a parameter-free model]. These models include the most commonly used network generator, the Niche model [@williamsSimpleRulesYield2000], as well as the original Cascade model [@cohenCommunityFoodWebs1990] and the derived Nested hierarchy model [@cattinPhylogeneticConstraintsAdaptation2004].
 
 These models are data light but there are some decisions that need to be made regarding what the expectations are on network structure.
 
 ### Species-specific networks
 
-In terms predicting interactions current models are rooted in feeding theory and allocate the links between species based on energy *e.g.,* diet models [@beckermanForagingBiologyPredicts2006; @petcheySizeForagingFood2008] have been used construct networks based on both profitability (as determined by the handling time, energy content, and predator attack rate) as well as abundance (prey density). [@woottonModularTheoryTrophic2023].
+These models capture the behaviour of the nodes by explicitly taking into account the properties of the different species in the community.
+
+Neutral networks are built on the assumption that foraging decisions are tied *only* to the abundance of species within the community [@pomeranzInferringPredatorPrey2019; @canardEmpiricalEvaluationNeutral2014; @krishnaNeutralnicheTheoryNestedness2008], and links are determined by the abundance of different species in the community.
```

Comment:
I guess if we want to go the species agnostic-specific spectrum then neutral models are actually species specific since abundance it at least some sort of 'trait' of a species (population)...